admin/src/scenes/settings/operations/actions/Inscription/ExportContactSimulation/ExportContactConvocation.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { HiOutlineInformationCircle } from "react-icons/hi";
+import { Button, Tooltip } from "@snu/ds/admin";
+import { CohortDto } from "snu-lib";
+import * as XLSX from "xlsx";
+import * as FileSaver from "file-saver";
+import { DepartmentService } from "@/services/departmentService";
+import { DepartmentServiceRoutes } from "snu-lib";
+import { useMutation } from "@tanstack/react-query";
+import { toastr } from "react-redux-toastr";
+
+interface ContactSimulationProps {
+  session: CohortDto;
+}
+
+export default function ExportContactConvocation({ session }: ContactSimulationProps) {
+  const [isInProgress, setIsInProgress] = useState(false);
+
+  const mutation = useMutation<DepartmentServiceRoutes["ExportContacts"]["params"]>({
```

Comment:
```suggestion
  const { mutate, isFetching } = useMutation<DepartmentServiceRoutes["ExportContacts"]["params"]>({
```
on peux récupérer directement le status de la requête dans le hook, pas besoin de faire un state `isInProgress`